content/blog/2017/08/2017-08-05-remoting-update.adoc

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "Remoting Update. Protocols deprecation, Java 8 requirement and plans"
+tags:
+- remoting
+- agents
```

Comment:
probably needs a generic "compatibility/upgrade" tag to indicate this is something requiring attention for people who do upgrades. Maybe it worth introducing the "core" tag as well after #1067 from @vkotovv and @rtyler